src/services/pipewire/node.hpp

```diff
@@ -86,10 +86,46 @@ class PwAudioChannel: public QObject {
 	/// including aux and custom channel ranges.
 	Q_INVOKABLE static QString toString(qs::service::pipewire::PwAudioChannel::Enum value);
 };
+///! The type of a pipewire node. Use bitwise comparisons to
+/// filter for audio, video, sink, source or stream nodes
+class PwNodeType: public QObject {
+	Q_OBJECT;
+	QML_ELEMENT;
+	QML_SINGLETON;
 
-enum class PwNodeType : quint8 {
-	Untracked,
-	Audio,
+public:
+	enum Enum : quint8 {
+		// A Pipewire node which is not being managed.
+		Untracked = 0x1,
```

Comment:
This should be 0x0, as for the others I've switched to using 0b for flags as its easier to verify.
e.g. `0b1`, `0b10`, `0b100`.